apriltag_ros/config/settings.yaml

```diff
@@ -1,11 +1,11 @@
 # AprilTag 3 code parameters
 # Find descriptions in apriltag/include/apriltag.h:struct apriltag_detector
 #                      apriltag/include/apriltag.h:struct apriltag_family
-tag_family:        'tag36h11' # options: tag36h11, tag25h9, tag16h5, tagStandard41h12
+tag_family:        'tag36h11' # options: tag36h11, tag25h9, tag16h5
 tag_threads:       2          # default: 2
 tag_decimate:      1.0        # default: 1.0
 tag_blur:          0.0        # default: 0.0
 tag_refine_edges:  1          # default: 1
 tag_debug:         0          # default: 0
 # Other parameters
-publish_tf:        true       # default: false
+publish_tf:        true       # default: true
```

Comment:
Default seems to be `false`.  See code here:
https://github.com/AprilRobotics/apriltag_ros/blob/d030c6d630088f36291c861ec8e39913347ee6cc/apriltag_ros/src/common_functions.cpp#L51